package.json

```diff
@@ -4,43 +4,125 @@
   "license": "ISC",
-  "dependencies": {
-    "npm-run-all": "^4.1.5"
-  },
+  "dependencies": {},
   "devDependencies": {
-    "@webdevstudios/css-coding-standards": "^1.0.1",
-    "@webdevstudios/js-coding-standards": "^1.1.1",
-    "@wordpress/scripts": "^12.1.1",
+    "@webdevstudios/js-coding-standards": "~1.3.0",
+    "@webdevstudios/npm-scripts": "~1.0.1",
+    "@wordpress/scripts": "~12.3.0",
     "copy-webpack-plugin": "^6.1.1",
     "cssnano": "^4.1.10",
+    "husky": "~4.3.0",
     "imagemin-webpack-plugin": "^2.4.2",
+    "lint-staged": "~10.4.1",
+    "npm-run-all": "~4.1.5",
+    "prettier": "~2.1.2",
+    "run-s": "0.0.0",
+    "stylelint": "~13.7.2",
+    "stylelint-config-wordpress": "~17.0.0",
     "svgstore-cli": "^2.0.0",
     "tailwindcss": "^1.8.0"
   },
   "eslintConfig": {
-    "extends": [
-      "plugin:@wordpress/eslint-plugin/esnext",
-      "plugin:@wordpress/eslint-plugin/react",
-      "@webdevstudios/js-coding-standards"
-    ],
+    "extends": "@webdevstudios/js-coding-standards",
     "ignorePatterns": [
-      "build/**",
-      "vendor/**"
+      "**/*.css",
+      "**/*.json",
+      "**/*.lock",
+      "**/*.php",
+      "**/*.png",
+      "**/*.scss",
+      "**/*.svg",
+      "*.min.js",
+      ".*dist.js",
+      ".*rc.js",
+      "./.*.js",
+      ".cache/**/*",
+      ".git*",
+      "build/**/*",
+      "cache/**/*",
+      "dist/**/*",
+      "node_modules/**/*",
+      "vendor/**/*"
+    ]
+  },
+  "stylelint": {
+    "extends": "stylelint-config-wordpress/scss",
+    "rules": {
+      "at-rule-no-unknown": null,
+      "block-no-empty": true,
+      "font-family-no-duplicate-names": true,
+      "font-family-no-missing-generic-family-keyword": true,
+      "function-url-quotes": "always",
+      "max-line-length": null,
+      "no-descending-specificity": null,
+      "no-duplicate-selectors": null,
+      "property-no-unknown": null,
+      "scss/at-rule-no-unknown": null,
+      "string-quotes": "single"
+    },
+    "ignoreFiles": [
+      "**/*.js",
+      "**/*.json",
+      "**/*.jsx",
+      "**/*.lock",
+      "**/*.php",
+      "**/*.png",
+      "**/*.svg",
+      "**/*.ts",
+      "**/*.tsx",
+      "*.min.js",
+      ".*dist.js",
+      ".*rc.js",
+      "./.*.js",
+      ".cache/**/*",
+      ".git*",
+      "build/**/*",
+      "cache/**/*",
+      "./style.css",
+      "node_modules/**/*",
+      "vendor/**/*"
     ]
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  },
+  "lint-staged": {
+    "*.js": "npm run lint:js",
+    "*.jsx": "npm run lint:js",
+    "*.ts": "npm run lint:js",
+    "*.tsx": "npm run lint:js",
+    "*.vue": "npm run lint:js",
+    "*.css": "npm run lint:styles",
+    "*.scss": "npm run lint:styles",
+    "*.php": "npm run lint:php",
+    "*.json": "prettier ./**/*.json --write",
+    "*.md": "prettier ./**/*.md --write",
+    "*.yml": "prettier ./**/*.yml --write"
+  },
+  "private": true,
+  "engines": {
+    "node": ">=12",
```

Comment:
More informative than helpful, the warning in .scripts is better. But a dev might see this or know this might be here.